docs/site/Loopback-component-authorization.md

```diff
@@ -8,48 +8,136 @@ permalink: /doc/en/lb4/Loopback-component-authorization.html
 
 ## Design
 
 LoopBack's highly extensible authorization package
 [@loopback/authorization](https://github.com/strongloop/loopback-next/tree/master/packages/authorization)
-provides various features and provisions to check access rights of a `Principal` on a API endpoint.
+provides various features and provisions to check access rights of a `Principal`
+on a API endpoint.
+
+It provides:
+
+- `a decorator` to annotate controller methods with authorization metadata
+  ([see, Configuring API Endpoints](##Configuring-API-Endpoints)) which consists
+  of the following
 
-It provides,
-- a decorator `to annotate controller methods` with authorization metadata ([see, Configuring API Endpoints](##Configuring-API-Endpoints)) which consists of the following
   - type of the protected resource (such as customer or order)
   - allowed roles and denied roles (to provide ACL based rules)
   - scopes (oauth scopes such as `delete public images` or `register user`)
-  - voters (supply a list of functions to vote on a decision)
-
-- an interceptor to enforce authorization with `user provided voters/authorizers` ([see, Programming Access Policies](##Programming-Access-Policies)) on API calls using,
-  - authorization metadata added by the decorator on the target controller method (roles, scopes, voters)
+  - voters (supply a list of functions to vote on a decision about a subject's
+    accessibility)
+
+- `an interceptor` to enforce authorization with user provided
+  voters/authorizers
+  ([see, Programming Access Policies](##Programming-Access-Policies)) on API
+  calls using,
+  - authorization metadata added by the decorator on the target controller
+    method (roles, scopes, voters)
   - a Principal/Subject deducted from the incoming request
   - [a decision matrix](##Authorization-by-decision-matrix)
 
+The expectations from various stake holders (LoopBack, Security analysts,
+Developers) for implementation of the authorization features are given below in
+the [Chain of Responsibility](##Chain-of-Responsibility) section.
+
 ![Authorization](./imgs/authorization.png)
 
+## Chain of Responsibility
+
+LoopBack as a framework provides certain provisions and expects the developers
+to extend with their specific implementations. Architects or Security analysts
+generally provide security policies to clarify how the developers should
+approach authorization.
+
+`The framework` provides,
 
-## Authorization Component
+- the `@authorize` decorator to provide authorization metadata and voters
+- various `types` and `interfaces` to declare user provided `artifacts`
+- a mechanism to enforce authorization policies
+  - abstractions of `authorizers` as user provided functions and voters
+  - create a decision matrix to combine results of all user provided
+    `authorizers`
+  - an interceptor which enforces policies by creating the decision matrix
+- a LoopBack authorization component which packs all the above
+
+`Architects` should,
```

Comment:
Cool have description for the role of `Architects` is better than `Users`

Reply:
thanks